runtime/lib/Project.gp

```diff
@@ -685,29 +685,25 @@ method removeClassFromPages Project aClass {
 
```

Comment:
This one shouldn't be merged but I don't know how to easily get rid of it. Still don't know how to use Git